src/hooks/init/plugin-install.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-process-exit */
+/* eslint-disable unicorn/no-process-exit */
+const { logger } = require('@twilio/cli-core').services.logging;
+const ALLOWED_ORGS = [
+  '@twilio/',
+  '@twilio-labs/'
+];
+
+function isTwilioPlugin(options) {
+  if (options.plugin.name === undefined) {
+    return false;
+  }
+  return ALLOWED_ORGS.find(function (org) {
+    return options.plugin.name.startsWith(org);
+  });
+}
+
+module.exports = async function (options) {
+  if (!isTwilioPlugin(options)) {
+    logger.warn('WARNING!!! You are attempting to install a plugin from an untrusted source.');
+    logger.warn('It could contain malicious software or in other ways compromise your system.');
+
+    const prompt = require('inquirer').prompt;
+    const response = await prompt([{
+      type: 'confirm',
+      name: 'continue',
+      message: 'Are you sure you want to continue?',
+      default: false
+    }]);
+    if (response.continue === false) {
+      process.exit(1);
```

Comment:
I think `this.exit(1)` works, just need to mock it in the tests like `ctx.exit = sinon.stub();`.

Reply:
`this.exit(1)` throws an error.

Reply:
I get `TypeError: this.exit is not a function`

Reply:
OK, mocking out ctx.exit appears to work. Just pushed the changes. 👍